src/Transport/Utils/BrokeredMessageConverter.cs

```diff
@@ -21,11 +21,17 @@ public static TransportMessage ToTransportMessage(this BrokeredMessage message)
                     headers[Headers.ReplyToAddress] = message.ReplyTo;
                 }
 
+                var intent = default(MessageIntentEnum);
```

Comment:
What would be a reasonable default? Should we set it specifically to a reasonable default?

Reply:
Can there be a default without knowing the real intent?...

Reply:
I think it would have to be Send/Publish. I don't think it ultimately matters once we get to the receiver